frontend/src/components/merchantHome/MerchantHome.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useState} from 'react';
+import {Box, Typography} from "@mui/material";
+import {Titles} from "../../common/constansts/titles";
+import PaymentEntry from "../paymentEntry/PaymentEntry";
+import GenerateQR from "../generateQR/GenerateQR";
+import "./styles.css";
+
+const MerchantHome = () => {
+    const [qrCode, setQrCode] = useState<string>("");
+
+    const updateQrCode = (qrCode: string) => {
+        setQrCode(qrCode);
+    }
+
+    return (
+        <Box className="merchant-home">
```

Comment:
client home component does not have this style. Keep consistency. All pages must look same